src/util/logs.rs

```diff
@@ -0,0 +1,64 @@
+use crate::subscriptions;
+use colored::Colorize;
+
+pub fn format_attr_log(mut log: subscriptions::deployment_logs::LogFields) {
+    if !log.attributes.is_empty() {
+        let mut timestamp: Option<String> = None;
+        let mut level: Option<String> = None;
+        let message = log.message;
+        let mut others = Vec::new();
+        // for some reason, not all have "" around the value
```

Comment:
Attributes are JSON, so `number` `null` `boolean` won't have quotes.

Reply:
i think it looked better when they all had it - cc @brody192